.github/workflows/planemo.yml

```
@@ -1,7 +1,13 @@
 name: Planemo
 on: [push, pull_request]
+
 env:
   GALAXY_BRANCH: release_22.05
```

That release brunch should be bumped. 

```suggestion
  GALAXY_BRANCH: release_23.1
```